operator/internal/manifests/storage/configure.go

```diff
@@ -193,6 +193,16 @@ func managedAuthCredentials(opts Options) []corev1.EnvVar {
 			}
 		}
 	case lokiv1.ObjectStorageSecretAzure:
+		if opts.OpenShift.ManagedAuthEnabled() {
+			return []corev1.EnvVar{
+				envVarFromSecret(EnvAzureStorageAccountName, opts.SecretName, KeyAzureStorageAccountName),
+				envVarFromSecret(EnvAzureClientID, opts.OpenShift.CloudCredentials.SecretName, "azure_client_id"),
+				envVarFromSecret(EnvAzureTenantID, opts.OpenShift.CloudCredentials.SecretName, "azure_tenant_id"),
+				envVarFromSecret(EnvAzureSubscriptionID, opts.OpenShift.CloudCredentials.SecretName, "azure_subscription_id"),
+				envVarFromValue(EnvAzureFederatedTokenFile, path.Join(azureTokenVolumeDirectory, "token")),
```

Comment:
nit. WDYT making the secret keys constants here?

Reply:
I'm already working on refactoring the configuration paths, volume(-mount) names and token paths, so that they can be the same across the different clouds as much as possible.

Reply:
Realized I misunderstood the comment because I only looked at the last line of the highlight. It's fixed now.